examples/reference/widgets/ChatEntry.ipynb

```diff
@@ -75,8 +77,73 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "pn.widgets.ChatEntry(\n",
-    "    value=pn.pane.Markdown(\"## Happy you're here!\"), user=\"Happy User\", avatar=\"😀\"\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "\n",
+    "data = np.random.rand(5, 2)\n",
+    "dataframe = pd.DataFrame(data, columns=list(\"AB\"))\n",
+    "\n",
+    "fig = plt.figure(figsize=(5,2.5))\n",
+    "ax = fig.add_subplot(111)\n",
+    "ax.plot(data)\n",
+    "plt.close(fig)\n",
+    "\n",
+    "pn.Column(\n",
+    "    ChatEntry(\n",
+    "        value=pn.widgets.Button(name=\"CLICK ME\", button_type=\"primary\"),\n",
```

Comment:
This is the example where the `Button` displays as `False`?

Reply:
I changed it from Parameter because of this; Parameter casts widgets into Str

Reply:
Is that a bug @philippjfr ?

Reply:
I've solved this by setting `ChatEntry._ignored_refs=('value',)`. Closing this one.